README.md

```diff
@@ -156,3 +157,197 @@ exit
 [jail]$ ping google.com # with the '-n' flag, network calls are not allowed
 ping: socket: Operation not permitted
 ```
+
+## Allowing Syscalls
+
+First step for allowing a new syscall, is discovering which syscall you need.
+
+Let's look at an example that uses `ping`. We see in this example that we get
+the message `Bad system call (core dumped)` when using the ping program.
+```
+(py310) [kyle@fedora capejail]$ sudo capejail -- bash
+[jail]# ping google.com
+Bad system call (core dumped)
```

Comment:
Is it possible to make the error say which system call was used?

Reply:
I’m not sure. One thing that makes that tricky is that the calling process that makes the syscall is immediately killed with SIGKILL, which is not recoverable. I’ll see if there’s any way to detect it from the capejail process though.